checker/tests/nullness/CastsNullness.java

```diff
@@ -88,9 +88,9 @@ void m() {
             // :: error: (assignment.type.incompatible)
             t = (@Nullable T) null;
             nt = (@Nullable T) null;
-            // :: warning: (cast.unsafe)
+            // :: error: (cast.incompatible)
```

Comment:
incompatible typecast `from "@FBCBottom @Nullable NullType" to "T[ extends @Initialized @NonNull Object super @Initialized @NonNull Void]"`

How is this incompatible?

Reply:
There's 2 qualifier comparisons, one is an upcast (`@FBCBottom to @Initialized`), another is a downcast (`@Nullable to @NonNull`). But, combined these two, it's neither a downcast nor an upcast.